### ISSUE TITLE
list-files fails for large files

### ISSUE THREAD

**Issue by tgpfeiffer** [retz/retz#81]
I have a large file (4.8GB) and when I try to get the details of this file via list-files, Retz fails with an HTTP 500 error. The standard output shows:

```
2016-11-14 16:47:56.926 ERROR [qtp856375702-30] spark.webserver.MatcherFilter -
com.fasterxml.jackson.databind.JsonMappingException: Numeric value (5122002944) out of range of int
...
{"gid":"1000","mode":"-rw-r--r--","mtime":1479109579.0,"nlink":1,"path":"\/var\/lib\/mesos\/...\/results\/99.bin","size":5122002944,"uid":"1000"}
...
```

I guess changing the respective value from Int to Long should fix the issue.

**Comment by kuenishi** [comment]
d1033598d27 will hopefully fix this

**Comment by tgpfeiffer** [comment]
Confirmed to be fixed on 0.0.33-SNAPSHOT.
